packages/utils/src/content-type.ts

```diff
@@ -1,24 +1,28 @@
 import fileType from 'file-type';
 import isSvg from 'is-svg';
-import * as mimeDB from 'mime-db';
+import mimeDB from './mime-db';
 import { parse, MediaType } from 'content-type';
 
-import { HttpHeaders } from '@hint/utils/dist/src/types/http-header';
-import { debug as d } from '@hint/utils/dist/src/debug';
-import { fileExtension as getFileExtension, fileName as getFileName } from '@hint/utils/dist/src/fs';
-import { normalizeString } from '@hint/utils/dist/src/misc/normalize-string';
-import { HTMLElement } from '@hint/utils/dist/src/dom/html';
+import { HttpHeaders } from './types/http-header';
+import { debug as d } from './debug';
+import { fileExtension as getFileExtension, fileName as getFileName } from './fs';
+import { normalizeString } from './misc/normalize-string';
 
 const debug = d(__filename);
 
+type HTMLElement = {
```

Comment:
This shouldn't be necessary, HTMLElement is now in the @hint/utils package.

Reply:
This should be fixed now!